support/mono_embeddinator.c

```diff
@@ -116,28 +142,14 @@ static GString* get_current_executable_path()
     DWORD ret = GetModuleFileNameA(hModule, pathbuf, MAX_PATH);
 
     return (ret > 0) ? g_string_new(pathbuf) : 0;
+#elif __ANDROID__
```

Comment:
Do we still use this given you override the path with `mono_embeddinator_set_assembly_path` now?